conf/test.config

```diff
@@ -48,37 +48,37 @@ params {
 
 profiles {
     annotation {
-        params.input               = "${baseDir}/tests/csv/3.0/vcf_single.csv"
+        params.input               = "${projectDir}/tests/csv/3.0/vcf_single.csv"
```

Comment:
I am fairly confused this hasn't caused any problems for us

Reply:
Same here

Reply:
yes, me too...